### DISCUSSION TITLE
Contribution rewards and game theory - In Draft

### DISCUSSION THREAD

**Discussion by stephen-rowan** [Quality-Assurance-DAO/QA-DAO-Legacy#36] [Tokenomics]
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br /><span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">Essay</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://stephen-rowan.github.io/" property="cc:attributionName" rel="https://stephen-rowan.github.io/">Stephen Whitenstall</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

### Ground Rules

An organization wants to reward all of its members.
This is a non-zero sum game in that the total gains and losses of all participants do not add up to zero.
This does not necessarily mean that everyone gains the same rewards or that no one can lose a reward.
This can either be a co-operative game or a non-co-operative game (regulated by rules).

### Dimension(s) of reward

A further consideration is whether a linear dimension of reward is a sufficient general representation of value in the organization. If contributions are only rewarded in terms of a fiat currency (fungible) then a member's contribution can only be valued in terms of that linear dimension (e.g. from 0 to N+1).

But value can be modeled in other dimensions besides financial wealth (fiat currency). For example in terms of health, reputation, voting rights etc.

### Equal Rewards

A simple non-zero sum game would require that everyone in the organization gets the same reward regardless of their contribution.

As a general system of payment for contribution this may be seen to be an unfair distribution of reward as it does not take account of differences in contribution.

But an Equal Reward element may be appropriate as a kind of representation of membership or dividend.

An Equal Reward Membership game might run continuously and pay out the same rewards to members on a periodic basis.

The Equal Reward Membership game could be the basis for some form of universal basic income within the organization. Income based simply on membership not contribution.

The Equal Reward Membership game could be the base level for a reputation system. If the same rewards were issued to all members in the form of a non-fungible token they might represent length of membership. Or a stake in membership itself that might be forfeited on leaving the organization (vesting periods).

**Comment by DanM3rcurius** [comment]
Great thinking 👏🏿 @stephen-rowan 
Equal rewards sounds like sth worth exploring further. Especially in the beginning this could be something simple enough to start with. 

*Possible?* the longer a membership is staked and the older the membership is, the higher chance to win a kind of lottery 
Later on reputation points and expertise accolades could weight in on the lottery chances or allow for access to additional rewards (in case a team or person excelled).
